src/test/java/frc/robot/filters/TestAutoYawFilter.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.filters;
+
+import org.junit.jupiter.api.Test;
+
+import edu.wpi.first.wpilibj.Timer;
+
+import static org.junit.Assert.assertEquals;
+
+import frc.utils.TestGyro;
+
+public class TestAutoYawFilter {
+    private static final double SMALLDIFFERENCE = 0.01;
+    
+    @Test
+    public void testAutoYawA() {
```

Comment:
what is 'test case 'A'? What senario is this?  A and B tells me nothing about how these are different, or what their intent is